Sources/Datadog/Core/Attributes/UserInfoProvider.swift

```diff
@@ -42,4 +41,6 @@ internal struct UserInfo {
     let name: String?
     let email: String?
     let extraInfo: [AttributeKey: AttributeValue]
+
+    internal static let empty = UserInfo(id: nil, name: nil, email: nil, extraInfo: [:])
```

Comment:
Nitpick: not sure how this is optimized by the compiler, but this convenience technically leads to a leak (never deallocated at least). An alternative would be an `internal convenience init`?

Reply:
True 👍. Replaced it with static computed `var` - same footprint as initializer, but keeps the verbosity.